src/gatling/resources/logback.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The `gatling` folder should be either inside `src/test` or (if java vs scala gives you trouble) at the root. 

Other services like rawls and leo have an `automation` folder at the root for automated integration tests.

Reply:
This can be done within ```build.gradle```. I think we should have an ```automation``` folder and place the ```gatling``` folder underneath it so that ```gatling``` tests can be separately deployed from unit tests.